packages/container/src/dom/createContainerMessageListener.ts

```diff
@@ -6,16 +6,15 @@ import {
   isValueChangeMessage,
   isAssetModalChangeMessage,
   RequestContext,
-  SetHeight,
   SetModalOpen,
   SetPluginReady,
-  SetValue,
+  SetContent,
   isGetContextMessage,
 } from '@storyblok/field-plugin'
 
 type ContainerActions = {
-  setHeight: SetHeight
-  setValue: SetValue
+  setHeight: (height: number) => void
```

Comment:
We should remove this `setHeight` too, right?

Reply:
Actually not, because the `setHeight` here is used by the Sandbox to change the height on incoming requests  https://github.com/storyblok/field-plugin/pull/180/files#diff-62705db1798835f98e756f45c86a641d92022824bc3fc4a7b11a0423cf72ce20R56